domains/image-dankamigos.json

```diff
@@ -0,0 +1,14 @@
+{
+  "description": "i wanted one more domain for testing my website ( other domain is dankamigos )",
+  "repo": "https://dankamigos.is-a.dev",
+  "owner": {
+    "username": "mokshdk",
+    "email": "anirudhsus001@gmail.com"
+  },
+  "record": {
+    "NS": [
+      "paul.ns.cloudflare.com",
+      "sydney.ns.cloudflare.com"
+    ]
+  }
```

Comment:
Reasoning does not suffice for the need for NS records as opposed to manually adding an additional record. Denied.